tests/test-cases.js

```diff
@@ -1711,6 +1711,30 @@ var UNPARSE_TESTS = [
 		config: { quotes: [true, false, true] },
 		expected: '"Col1",Col2,"Col3"\r\n"a",b,"c"\r\n"d",e,"f"'
 	},
+	{
+		description: "Force quotes around string fields only",
+		input: [['a', 'b', 'c'], ['d', 10, true]],
+		config: { quotes: function(value) { return typeof value === 'string'; } },
+		expected: '"a","b","c"\r\n"d",10,true'
+	},
+	{
+		description: "Force quotes around string fields only (with header row)",
+		input: [{ "Col1": "a", "Col2": "b", "Col3": "c" }, { "Col1": "d", "Col2": 10, "Col3": true }],
+		config: { quotes: function(value) { return typeof value === 'string'; } },
+		expected: '"Col1","Col2","Col3"\r\n"a","b","c"\r\n"d",10,true'
+	},
+	{
+		description: "Force quotes around values in first column only",
```

Comment:
I think those test are redundant as they test the same functionality but using a diferent function. 

Could you please remove them and include only the first two. 

Limiting the number of required test functions reduces the maintentance cost when refactoring the aplication.

Reply:
They are redundant because the tested functionality is exactly the same, what only changes is the quotes function which is an user parameter which we do not care about it. Our code should work for all the user provided functions. 

So please, include only the first two test.